src/xenia/kernel/xboxkrnl/xboxkrnl_memory.cc

```diff
@@ -101,6 +101,10 @@ dword_result_t NtAllocateVirtualMemory(lpdword_t base_addr_ptr,
   if (*base_addr_ptr != 0) {
     // ignore specified page size when base address is specified.
     auto heap = kernel_memory()->LookupHeap(*base_addr_ptr);
+
```

Comment:
Please balance the new lines, no need for new lines between getting the heap and checking if it's usable.